presto-main/src/test/java/io/prestosql/sql/query/TestSubqueries.java

```diff
@@ -87,6 +87,20 @@ public void testUnsupportedSubqueriesWithCoercions()
                 UNSUPPORTED_CORRELATED_SUBQUERY_ERROR_MSG);
     }
 
+    @Test
+    public void testSimpleArraySubqueries()
```

Comment:
Super simple test - will of course add many more once I get the basic flow working.